oca_dependencies.txt

```diff
@@ -1,8 +1,10 @@
 # Add account-financial-tools repo dependency to have access to currency_rate_update module.
 # Add account-payment repo dependency to have access to account_vat_on_payment and account_voucher_cash_basis modules.
+# Add bank-statement-import repo dependency to have access to MT940 import statements modules.
 # Add hr repo dependency to have access to some hr modules.
 # Add partner-contact repo dependency to have access to partner_create_by_vat module.
 account-financial-tools https://github.com/OCA/account-financial-tools 8.0
 account-payment https://github.com/OCA/account-payment 8.0
+bank-statement-import https://github.com/OCA/bank-statement-import 8.0
```

Comment:
The file can be extremely simplified in this case, as the organization and branch is the same:
```
bank-statement-import
hr
partner-contact
...
```